src/iceflow/ingest/models.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from typing import Generic, TypeVar
+
+import pandera as pa
+from pandera.typing import DataFrame, Index, Series
+
+from iceflow.itrf import SUPPORTED_ITRFS
+
+TDataFrame_co = TypeVar("TDataFrame_co", covariant=True)
```

Comment:
The `_co` suffix could mean covariant or contravariant. IMO, spell out all the way!

Reply:
https://peps.python.org/pep-0484/#covariance-and-contravariance

> By convention, it is recommended to use names ending in _co for type variables defined with covariant=True and names ending in _contra for that defined with contravariant=True.

Reply:
yay AI! :laughing: . Agreed about being being verbose though. At a minimum, if I go this route, I'll link to the docs.

Reply:
`ruff` configured to complain about this:

```
ruff.....................................................................Failed
- hook id: ruff
- exit code: 1

src/iceflow/ingest/models.py:10:24: PLC0105 `TypeVar` name "TDataFrame_covariant" does not reflect its covariance; consider renaming it to "TDataFrame_covariant_co"
   |
 8 | from iceflow.itrf import SUPPORTED_ITRFS
 9 |
10 | TDataFrame_covariant = TypeVar("TDataFrame_covariant", covariant=True)
   |                        ^^^^^^^ PLC0105
   |

Found 1 error.
```

I love its suggestion:

> name "TDataFrame_covariant" does not reflect its covariance; consider renaming it to "TDataFrame_covariant_co"

Reply:
5ffdfd1 * origin/e2e-test Add note about `_co` convention